src/components/ActivityLogPreview.svelte

```diff
@@ -2,7 +2,8 @@
 </script>
 
 <!-- TODO: Connect activities -->
-<div class="min-w-[300px] border-l border-divider pl-5">
+<div></div>
+<!-- <div class="min-w-[300px] border-l border-divider pl-5">
```

Comment:
can this be removed rather than commented out?

Reply:
In the next story, we're using this code, so keeping it in.